src/peft/tuners/lora/config.py

```diff
@@ -117,6 +119,25 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    if is_megatron_available():
+        megatron_config: Optional[ModuleType] = field(
+            default=None,
+            metadata={
+                "help": (
+                    "The TransformerConfig of Megatron, it used to create lora's parallel linear. "
+                    "You can get it like this, `core_transformer_config_from_args(get_args())`"
```

Comment:
Let's add that the two functions are from megatron.

Reply:
"The TransformerConfig from Megatron" like this?

Reply:
> "The TransformerConfig from Megatron" like this?

"The TransformerConfig from Megatron" like this?

Reply:
> Let's add that the two functions are from megatron.

"The TransformerConfig from Megatron" like this?

Reply:
Sorry, I meant to refer to `core_transformer_config_from_args` and `get_args`

Reply:
> Sorry, I meant to refer to `core_transformer_config_from_args` and `get_args`

Oh, ok. Now is like this:
```python
            metadata={
                "help": (
                    "The TransformerConfig from Megatron, it is used to create LoRA's parallel linear. "
                    "You can get it like this, `core_transformer_config_from_args(get_args())`, this two functions are from megatron. "
                )
            },
```